File: Syeew/ClientApp/src/app/models/interfaces/ICompany.ts
import { Dimention } from "../Dimention";
import { ManegementSystem } from "../ManegementSystem";
import { TypeOfCompany } from "../TypeOfCompany";
import { IQuantitativeData } from "./IQuantitativeData";

export interface ICompany{

  ragioneSociale: string;

  nomeAttivita: string;

  tipoAttivita: string;

  gestionale: string;

  dimensioneFatturato: string;

  dimensioneAddetti: string;

  indirizzo: string;

  citta: string;

  provincia: string;

  datas: IQuantitativeData[];

}

File: DataSourceSyeew/Entities/InterfacesEntities/ICompany.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataSourceSyeew.Entities.InterfacesEntities
{
    public interface ICompany
    {
        string RagioneSociale { get; set; }
        string NomeAttivita { get; set; }
        string TipoAttivita { get; set; }

        string Gestionale { get; set; }
        string DimensioneFatturato { get; set; }
        string DimensioneAddetti { get; set; }

        string Indirizzo { get; set; }
        string Citta { get; set; }
        string Provincia { get; set; }

        ICollection<QuantitativeData> Datas { get; set; }
    }
}
